Haptics/app/src/main/res/values/strings.xml

```diff
@@ -17,4 +17,19 @@
     <string name="app_name">Haptic Sampler</string>
     <string name="drawer_content_description">Drawer menu</string>
     <string name="home_screen_title">Vibration effects</string>
+    <string name="home_screen_predefined_effects">Predefined effects</string>
+    <string name="home_screen_tick">Tick</string>
+    <string name="home_screen_click">Click</string>
+    <string name="home_screen_heavy_click">Heavy click</string>
+    <string name="home_screen_double_click">Double click</string>
+    <string name="home_screen_feedback_constants">Haptic feedback constants</string>
+    <string name="home_screen_submit_success">Submit success</string>
```

Comment:
Maybe we should rename to CONFIRM and REJECT, to match the code APIs.

Reply:
Done